app/src/main/res/values-de/strings.xml

```diff
@@ -132,18 +132,18 @@
     <string name="msg_file_description">Datei Beschreibung</string>
     <string name="msg_send">Sende</string>
     <string name="msg_sent_attachment">Sende Anhang</string>
-    <string name="msg_welcome_to_rocket_chat">Welcome to Rocket.Chat</string> <!-- TODO Add translation -->
-    <string name="msg_team_communication">Open Source Communication</string> <!-- TODO Add translation -->
-    <string name="msg_login_with_email">Login with <b>e-mail</b></string> <!-- TODO Add translation -->
-    <string name="msg_create_account">Create an account</string> <!-- TODO Add translation -->
-    <string name="msg_continue_with_facebook">Continue with <b>Facebook</b></string> <!-- TODO Add translation -->
-    <string name="msg_continue_with_github">Continue with <b>Github</b></string> <!-- TODO Add translation -->
-    <string name="msg_continue_with_google">Continue with <b>Google</b></string> <!-- TODO Add translation -->
-    <string name="msg_continue_with_linkedin">Continue with <b>Linkedin</b></string> <!-- TODO Add translation -->
-    <string name="msg_continue_with_gitlab">Continue with <b>GitLab</b></string> <!-- TODO Add translation -->
-    <string name="msg_continue_with_wordpress">Continue with <b>WordPress</b></string> <!-- TODO Add translation -->
-    <string name="msg_two_factor_authentication">Two-factor Authentication</string> <!-- TODO Add translation -->
-    <string name="msg__your_2fa_code">What’s your 2FA code?</string> <!-- TODO Add translation -->
+    <string name="msg_welcome_to_rocket_chat">Wilkommen bei Rocket.Chat</string>
+    <string name="msg_team_communication">Open Source Kommunikation</string>
+    <string name="msg_login_with_email">Mit <b>e-mail</b> einloggen</string>
+    <string name="msg_create_account">Einen Account erstellen</string>
+    <string name="msg_continue_with_facebook">Mit <b>Facebook</b> einloggen</string>
```

Comment:
Is `Mit` the translate to `Login` in English here? If so, could you translate it to the `Continue` in English?